webapp/components/PasswordReset/Request.vue

```diff
@@ -68,8 +69,11 @@ export default {
     }
   },
   computed: {
+    email() {
+      return normalizeEmail(this.formData.email)
+    },
     submitMessage() {
-      const { email } = this.formData
+      const { email } = this
```

Comment:
I think this is the graphql mutation which has Boolean as return type. Thus we cannot take the normalized email from the backend but have to normalize it client-wise.